avocado/core/safeloader.py

```diff
@@ -30,36 +30,58 @@ class AvocadoModule(object):
     """
     Representation of a module that might contain avocado.Test tests
     """
-    __slots__ = 'path', 'test_imports', 'mod_imports', 'mod'
+    __slots__ = 'path', 'test_imports', 'mod_imports', 'mod', 'imported_objects'
```

Comment:
it's not used anywhere, which is why I made it part of the big commit where it is used. Anyway it's not breaking anything and hopefully will be merged together with the actual change, so let's split it...